guides/common/modules/proc_adding-azure-connection.adoc

```diff
@@ -13,6 +13,7 @@ Use this procedure to add Microsoft Azure Resource Manager as a compute resource
 . In the *Client Secret* field, enter the client secret.
 . In the *Subscription ID* field, enter the subscription ID.
 . In the *Tenant ID* field, enter the Directory or _tenant_ ID.
+. Click *Load Regions*. This tests if your connection to the Azure Resource Manager was successful, and accordingly will load the available regions specific to your subscription.
```

Comment:
```suggestion
. Click *Load Regions*. This tests if your connection to Azure Resource Manager is successful and loads the regions available in your subscription.
```
Keep things in present tense for simplicity. Remove article before a product name. Remove fluff.

Reply:
Done